_docs/ops/runbook/troubleshooting-logsearch.md

```diff
@@ -67,7 +70,8 @@ cp /var/vcap/jobs/ingestor_syslog/config/logstash.conf /var/vcap/data/ingestor_s
 
 Edit the `/tmp/logstash-restore.conf` and make the following changes:
 
-#### Update the "input" block to only contain s3
+#### Update the "input" block to only contain S3 information
```

Comment:
If you want to run multiple days, hours or minutes, you can declare multiple s3 blocks with the unique prefix paths (ie `2021/01/00, 2021/01/01, 2021/01/02/00`) to run multiple batches of data without needing to rerun and update the s3 block settings.  Each block just needs a unique prefix and unique `.sincedb` file

Reply:
oh, yeah, this was wonderful once we figured it out!

Reply:
Will add it in, thank you!

Reply:
Done, see below!